workbench/src/renderer/app.jsx

```diff
@@ -200,9 +200,8 @@ export default class App extends React.Component {
   async updateInvestList() {
     const coreModels = {};
     const investList = await getInvestModelNames();
```

Comment:
Should `getInvestModelNames` be `getInvestModelIds`?

Reply:
Sure, I renamed all these